script.js

```diff
@@ -0,0 +1,65 @@
+const booksData = [];
+
+function removebook(bookId) {
+  let storedData = JSON.parse(localStorage.getItem('storedData'));
+  storedData = storedData.filter((bookobj) => {
+    if (bookobj.id.toString() === bookId) {
+      return false;
+    }
+    return true;
+  });
+  localStorage.setItem('storedData', JSON.stringify(storedData));
+}
+
+function createBook(book) {
+  const booksCont = document.getElementById('booksCont');
+  const bookDiv = document.createElement('div');
+  bookDiv.className = 'book';
+  bookDiv.id = `book-${book.id}`;
+  const bookElement = `
+        <h4 class="bookTitle">${book.title}</h4>
+        <p class="bookAuthor">${book.author}</p>
+        <button class="removeBtn">Remove</button>
+        <hr>
+      `;
+  bookDiv.innerHTML += bookElement;
+  bookDiv.querySelector('.removeBtn').addEventListener('click', (event) => {
+    const book = document.getElementById(event.target.parentElement.id);
+    book.parentElement.removeChild(book);
+    const bookId = book.id.substring(5, book.id.length);
+    removebook(bookId);
+  });
+  booksCont.append(bookDiv);
+}
+
+function storeInputData() {
+  const inputData = {
+    title: document.getElementById('title').value,
+    author: document.getElementById('author').value,
+    id: Math.floor(Math.random() * 10000),
+  };
+  const storedData = JSON.parse(localStorage.getItem('storedData'));
+  storedData.push(inputData);
```

Comment:
Well done on your implementation! However, I'd like to suggest a slight improvement that could enhance your code quality.
- [ ] [Optional]  Currently, you are using the `array.push()` method to add new elements to an array. While this approach works well, I recommend considering the use of the `array.concat()` method instead. The [concat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) method of [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) instances allows you to merge two or more arrays. The key advantage here is that it doesn't modify the existing arrays; instead, it returns a new array that combines the contents of the original arrays. This can help maintain data [immutability](https://developer.mozilla.org/en-US/docs/Glossary/Immutable) and prevent unexpected side effects.

By using `array.concat()` in your context, you'll be creating a new array that includes your newly generated book while keeping the original array intact. This can contribute to a more robust and maintainable codebase. Keep up the excellent work, and happy coding! 🌟📝🚀"